### DISCUSSION TITLE
integrate ArcGIS，layer is not attached to the ground

### DISCUSSION THREAD

**Discussion by endeep** [visgl/deck.gl#8144] [Q&A]
i use the example project (https://github.com/visgl/deck.gl/tree/8.9-release/examples/website/i3s),
when i set basemap to "satellite" or other, layer ( Tile3DLayer ) is not attached to the ground

`  map: new ArcGISMap({
          basemap: "satellite",
          // basemap: "dark-gray-vector",
        }) 
`

png 1 dark-gray-vector  is normal
png 2 satellite  is not attached to the ground

![1](https://github.com/visgl/deck.gl/assets/18085918/c0db7dd6-b9af-4601-a09d-bac660d4670c)
![2](https://github.com/visgl/deck.gl/assets/18085918/b2190e23-6782-428f-b314-ba5a55bca423)

**Comment by Pessimistress** [comment]
That is expected. 3D tiles are rendered at their real-world altitude. This is baked into the tiles themselves.
